R/resample.R

```diff
@@ -73,10 +75,11 @@
 #'   measures = list(mmce, setAggregation(mmce, train.mean)))
 #' print(r$aggr)
 resample = function(learner, task, resampling, measures, weights = NULL, models = FALSE,
-  extract, keep.pred = TRUE, ..., show.info = getMlrOption("show.info")) {
+  extract, keep.pred = TRUE, cache = FALSE, ..., show.info = getMlrOption("show.info")) {
```

Comment:
I would make `cache` an argument to the functions that create the filter wrappers, not to `resample`. I think this is misleading in that in most cases this won't do anything because nothing can be cached.

Reply:
I tried this at first but faced errors when calling `resample()` then stating that `cache` was not found etc pp.

Reply:
Hmm, I would investigate that further. If the cache isn't found in some cases it may indicate that not as much caching is done as you may think.